dev-util/electron/electron-0.36.9.ebuild

```diff
@@ -0,0 +1,707 @@
+	if tc-is-cross-compiler; then
+		export AR_host=$(tc-getBUILD_AR)
+		export CC_host=$(tc-getBUILD_CC)
+		export CXX_host=$(tc-getBUILD_CXX)
+		export NM_host=$(tc-getBUILD_NM)
+	fi
+
+	# Bug 491582.
+	export TMPDIR="${WORKDIR}/temp"
+	mkdir -p -m 755 "${TMPDIR}" || die
+
+	if ! use system-ffmpeg; then
+		local build_ffmpeg_args=""
+		if use pic && [[ "${ffmpeg_target_arch}" == "ia32" ]]; then
+			build_ffmpeg_args+=" --disable-asm"
+		fi
+
+		# Re-configure bundled ffmpeg. See bug #491378 for example reasons.
+		einfo "Configuring bundled ffmpeg..."
+		pushd third_party/ffmpeg > /dev/null || die
+		chromium/scripts/build_ffmpeg.py linux ${ffmpeg_target_arch} \
+			--branding ${ffmpeg_branding} -- ${build_ffmpeg_args} || die
+		chromium/scripts/copy_config.sh || die
+		chromium/scripts/generate_gyp.py || die
+		popd > /dev/null || die
+	fi
+
+	third_party/libaddressinput/chromium/tools/update-strings.py || die
+
+	einfo "Configuring bundled nodejs..."
+	pushd vendor/node > /dev/null || die
+	# Make sure gyp_node does not run
+	echo '#!/usr/bin/env python' > tools/gyp_node.py
+	./configure --shared-openssl --shared-libuv --shared-http-parser \
+				--shared-zlib --without-npm --with-intl=system-icu \
+				--without-dtrace --dest-cpu=${target_arch} \
+				--prefix="" || die
+	popd > /dev/null || die
+
+	# libchromiumcontent configuration
+	myconf+=" -Dcomponent=static_library"
+	myconf+=" -Dmac_mas_build=0"
+	myconf+=' -Dicu_small="false"'
+	myconf+=" -Dlibchromiumcontent_component=0"
+	myconf+=" -Dlibrary=static_library"
+	myconf+=" -Dmas_build=0"
+
+	einfo "Configuring electron..."
+	build/linux/unbundle/replace_gyp_files.py ${myconf} || die
+
+	myconf+=" -Ivendor/node/config.gypi
+			  -Icommon.gypi
+			  atom.gyp"
+
+	egyp_chromium ${myconf} || die
+}
+
+eninja() {
+	if [[ -z ${NINJAOPTS+set} ]]; then
+		local jobs=$(makeopts_jobs)
+		local loadavg=$(makeopts_loadavg)
+
+		if [[ ${MAKEOPTS} == *-j* && ${jobs} != 999 ]]; then
+			NINJAOPTS+=" -j ${jobs}"
+		fi
+		if [[ ${MAKEOPTS} == *-l* && ${loadavg} != 999 ]]; then
+			NINJAOPTS+=" -l ${loadavg}"
+		fi
+	fi
+	set -- ninja -v ${NINJAOPTS} "$@"
+	echo "$@"
+	"$@"
+}
+
+src_compile() {
+	local ninja_targets="electron"
+
+	# Even though ninja autodetects number of CPUs, we respect
+	# user's options, for debugging with -j 1 or any other reason.
+	eninja -C out/R ${ninja_targets} || die
+}
+
+src_install() {
+	local install_dir="$(_get_install_dir)"
+	local install_suffix="$(_get_install_suffix)"
+	local LIBDIR="${ED}/usr/$(get_libdir)"
+
+	pushd out/R/locales > /dev/null || die
+	chromium_remove_language_paks
+	popd > /dev/null || die
+
+	# Install Electron
+	insinto "${install_dir}"
+	exeinto "${install_dir}"
+	doexe out/R/electron
+	doins out/R/libv8.so
+	doins out/R/libnode.so
+	fperms +x "${install_dir}/libv8.so" "${install_dir}/libnode.so"
+	doins out/R/content_shell.pak
+	doins out/R/natives_blob.bin
+	doins out/R/snapshot_blob.bin
+	rm -rf out/R/resources/inspector
```

Comment:
Probably just `rm -r` instead of `rm -rv`; we probably don't need rm to print every file it is deleting.